zanata-war/src/main/java/org/zanata/util/CommonMarkRenderer.java

```diff
@@ -21,20 +21,22 @@
 package org.zanata.util;
 
 import com.google.common.base.Throwables;
+import com.google.common.io.Resources;
+import jdk.nashorn.api.scripting.JSObject;
```

Comment:
Nashorn is part of both Oracle JDK and OpenJDK 1.8.  It's unfortunate that we have to depend on a Nashorn interface here, but there seems to be no way around that if we want the best performance.